apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/LoadedFilterManager.java

```diff
@@ -341,29 +375,30 @@ public static LoadedFileFilter buildFilter(
   /* DB Operations */
 
   /**
-   * Return the max date from the LoadedBatch table
+   * Return the max date from the LoadedBatch table. If no batches are present, then the schema
+   * migration time which will be a timestamp before the first loaded batch
    *
    * @return the max date
    */
-  private Date fetchLastLoadedBatchTime() {
+  private Optional<Date> fetchLastLoadedBatchCreated() {
     Date maxCreated =
         entityManager
             .createQuery("select max(b.created) from LoadedBatch b", Date.class)
             .getSingleResult();
-    return Optional.ofNullable(maxCreated).orElse(new Date());
+    return Optional.ofNullable(maxCreated);
   }
```

Comment:
I've moved this else clause out of this function to the caller for code readability reasons. This logic shouldn't handle the case when the table is empty.